rosdep/python.yaml

```
@@ -2493,6 +2493,23 @@ python-levenshtein:
   fedora: [python-Levenshtein]
   gentoo: [dev-python/python-levenshtein]
   ubuntu: [python-levenshtein]
+python3-pylibdmtx:
+  debian: [python3-pylibdmtx]
+  fedora:
+    pip:
+      packages: [pylibdmtx]
+  osx:
+    pip:
+      packages: [pylibdmtx]
+  ubuntu:
+    "*":
```

```suggestion
    '*':
```

(otherwise the linters will complain)